packages/@n8n/nodes-langchain/nodes/agents/Agent/agents/ConversationalAgent/execute.ts

```diff
@@ -31,9 +31,11 @@ export async function conversationalAgentExecute(
 		| BaseChatMemory
 		| undefined;
 
-	const tools = await getConnectedTools(this, nodeVersion >= 1.5);
+	const tools = await getConnectedTools(this, nodeVersion >= 1.5, true);
```

Comment:
`true` is a default value for the 3rd argument here, it might be omitted

Reply:
Ah, true. Was just testing something and forgot to revert this. Thanks!